src/modules/Messages/store/store.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { getUnreadMessagesCount } from './actions';
+import { MessagesState } from './types';
+
+const initialState: MessagesState = {
+  unreadMessages: {
+    count: 0,
+    status: 'uninitialised',
```

Comment:
I would recommend to create enum in common/constants. What do you think?
`
 StatusEnum {
  Uninitialized: 'uninitialized',
  Fulfilled: 'fulfilled',
  ...
}
`

uninitialized